firebase-database/firebase-database.gradle.kts

```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+plugins { id("firebase-library") }
+
+firebaseLibrary {
+  testLab.enabled = true
+  publishSources = true
+}
+
+android {
+  val targetSdkVersion: Int by rootProject
+  val minSdkVersion: Int by rootProject
+
+  installation.timeOutInMs = 60 * 1000
+  compileSdk = targetSdkVersion
+
+  defaultConfig {
+    minSdk = minSdkVersion
+    targetSdk = targetSdkVersion
+    multiDexEnabled = true
+    testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+  }
+
+  buildTypes { release { isMinifyEnabled = false } }
+
+  sourceSets {
+    named("androidTest") { java.srcDir("src/testUtil/java") }
+    named("test") { java.srcDir("src/testUtil") }
+
+    compileOptions {
+      sourceCompatibility = JavaVersion.VERSION_1_8
+      targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    packagingOptions.resources.excludes += "META-INF/DEPENDENCIES"
+    testOptions.unitTests.isIncludeAndroidResources = true
+  }
+}
+
+dependencies {
+  implementation("com.google.firebase:firebase-common:20.3.1")
+  implementation("com.google.firebase:firebase-components:17.1.0")
+  implementation("com.google.firebase:firebase-appcheck-interop:16.1.1")
+  implementation("com.google.firebase:firebase-database-collection:18.0.1")
+
+  implementation("androidx.annotation:annotation:1.1.0")
+  implementation("com.google.android.gms:play-services-base:18.0.1")
+  implementation("com.google.android.gms:play-services-tasks:18.0.1")
+  implementation("com.google.android.gms:play-services-basement:18.1.0")
```

Comment:
These also exist in the version catalog

Reply:
Since it depends on common, and common depends on 

```
    implementation(libs.playservices.basement)
    implementation(libs.playservices.tasks)
```

those are already in the latest version.

So the only question would be base.

Reply:
Hmm, I'll run some tests tomorrow and peep if upgrading everything breaks anything (or find out what _we can_ upgrade). If not, it'll just be a minor bump anyways; but I'll ping db folk to make sure they're alright with it. Thoughts?

Reply:
I think it's OK to just bump those, and then wait until database has a meaningful change to do the release. 

The last release of DB is [20.1.0](https://maven.google.com/web/index.html?q=firebase-co#com.google.firebase:firebase-database:20.1.0) and it depends in `firebase-common:20.2.0`,  which is outdated anyway.

Reply:
In case you want to take a look, here's the list of deps from a kotlin app that only depends on db

https://gist.github.com/rlazo/55e8fc9f331dfb4083e6ad0b2bb9f809

Reply:
SGTM :)